### DISCUSSION TITLE
Cors error on netlify

### DISCUSSION THREAD

**Discussion by cambo2015** [appwrite/appwrite#1239] [Q&A] [answered]
Hi again! Appwrite is awesome! I was able to get appwrite installed via the digital ocean marketplace and on my local machine. 

I am stuck at a cors error though. I know what it means. How do I get appwrite's CORS policy to allow the netlify react app? Is there a CORS env var in the docker file to recognize the netlify domain?  I did register the web app in the appwrite project dashboard, but no  luck. Thanks again

**Comment by Duramax2018** [comment]


**Reply by cambo2015** [reply]
Hey no hacked email here. I'm just trying to learn appwrite. It's a wonderful community. Any answers are appreciated.

**Comment by cambo2015** [comment, marked as answer]
I found my own answer and maybe this can help others. When you select add web platform in the dashboard, make sure to leave out the https:// . This at least worked for my temporary netlify domain. Hope this helps someone.

**Reply by eldadfux** [reply]
Just to confirm, the project console requires you to input the hostname, not the URL as seen below.

![image](https://user-images.githubusercontent.com/1297371/120890602-8fd5e900-c60c-11eb-8f23-1f3439d20f85.png)
